devops/github_issue_alert/program.py

```diff
@@ -0,0 +1,38 @@
+"""Monitors GitHub issue creation and comment activity.
+
+Sending the notifications to Slack to keep teams updated.
+"""
+
+import os
+
+from autokitteh.slack import slack_client
+
+
+SLACK_CHANNEL = os.getenv("SLACK_CHANNEL", "")
+
+slack = slack_client("slack_conn")
+
+
+def on_issue_comment(event):
+    comment_body = event["data"]["comment"]["body"]
```

Comment:
1. Define `data = event.data` as a helper variable
2. Use `data.foo.bar` instead of `data["foo"]["bar"]` because these fields are guaranteed to exist, and it's easier to read that code
3. When you use `data.foo.bar` it's not clear to me that it's useful to define all these helper variables instead of just using them in the f-strings, only `comment_author` and `issue_body` seem like good helper variables
4. Shorter var names are better, when they make sense: `author` is clear enough and shorter than `comment_author`